examples/glyph-lcd-cairo.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+#
+#  pycairo/cairocffi-based glyph-lcd example - Copyright 2017 Hin-Tak Leung
+#  Distributed under the terms of the new BSD license.
+#
+#  rewrite of the numply,matplotlib-based example from Nicolas P. Rougier
+#  - Not immitating the upside-downness
+#  - Do LCD_V side by side also
+#
+#  Limitation: Suface.get_data() is not in the "python 3, pycairo < 1.11" combo.
+#
+# -----------------------------------------------------------------------------
+'''
+Glyph bitmap LCD display (sub-pixel) rendering
+'''
+from freetype import *
+# use Matrix() from Cairo instead of from Freetype
+from cairo import Context, ImageSurface, FORMAT_ARGB32, SurfacePattern, FILTER_BEST, Matrix
+
+if __name__ == '__main__':
+    from PIL import Image
+    from numpy import ndarray, ubyte, uint32
+
+    face = Face('./Vera.ttf')
+    face.set_char_size( 48*64 )
+    face.load_char('S', FT_LOAD_RENDER |
+                        FT_LOAD_TARGET_LCD )
+    bitmap = face.glyph.bitmap
+    width  = face.glyph.bitmap.width//3
+    rows   = face.glyph.bitmap.rows
+    pitch  = face.glyph.bitmap.pitch
+    whole_thing = (c_ubyte * (pitch * rows))()
+    memmove(pointer(whole_thing), bitmap._FT_Bitmap.buffer,
+            pitch * rows)
+
+    I = ImageSurface(FORMAT_ARGB32, width, rows)
+    ndR = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=0,
+                  strides=[pitch, 3])
+    ndG = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=1,
+                  strides=[pitch, 3])
+    ndB = ndarray(shape=(rows,width), buffer=whole_thing,
+                  dtype=ubyte, order='C',
+                  offset=2,
+                  strides=[pitch, 3])
+    ndI = ndarray(shape=(rows,width), buffer=I.get_data(),
+                  dtype=uint32, order='C',
+                  strides=[I.get_stride(), 4])
+    ndI[:,:] = 4278190080 + ndR[:,:] * 65536 + ndG[:,:] * 256 + ndB[:,:]
```

Comment:
I'd add a comment next to it saying what those numbers are, best of both worlds?